index.js

```diff
@@ -255,8 +255,12 @@ Funnel.prototype.build = function() {
     /*eslint-enable no-lonely-if*/
 
     this._isRebuild = true;
-  } else {
+  } else if (inputPathExists) {
```

Comment:
should we inline `fs.existsSync(inputPath);` here to avoid needless IO?

Reply:
Yeah, @rwjblue and I looked at that and I *think* it ends up being a required check given the new structure here: it has to be executed regardless of which path is taken.

Reply:
Currently it is only being used here, will it be used elsewhere?

Reply:
@stefanpenner 	yes it is being hoisted from 

https://github.com/broccolijs/broccoli-funnel/blob/bfea8d4c79e624b02a248efdfa345e5055fbc0ea/index.js#L219 (inside the first block of this `if`) so that we can share it both inside that block and this `else if`. This is not adding an extra I/O operation AFAICT.

Reply:
It's used throughout the other `if` branch as well; see lines [229] and [242]

[229]: https://github.com/broccolijs/broccoli-funnel/pull/106/files/a76c4ee50cf2b39d7d28d5c23f957447a7704756#diff-168726dbe96b3ce427e7fedce31bb0bcR229
[242]: https://github.com/broccolijs/broccoli-funnel/pull/106/files/a76c4ee50cf2b39d7d28d5c23f957447a7704756#diff-168726dbe96b3ce427e7fedce31bb0bcR242